plugins/org.komodo.core/src/org/komodo/repository/ObjectImpl.java

```diff
@@ -583,6 +583,16 @@ public KomodoObject getChild( final UnitOfWork transaction,
     /**
      * {@inheritDoc}
      *
+     * @see org.komodo.spi.repository.KomodoObject#getChildTypes()
+     */
+    @Override
+    public KomodoType[] getChildTypes() {
+        return KomodoType.NO_TYPES;
+    }
```

Comment:
Not sure but this feels wrong. Just because an object has not been relationally defined should not discount the fact it has children. Can this work a little harder to maybe resolve itself then return the resolved object's getChildTypes(). Otherwise return KomodoType.UNKNOWN maybe?